src/app/core/services/basic-rundown-state.service.ts

```diff
@@ -61,6 +61,7 @@ export class BasicRundownStateService implements OnDestroy {
   private subscribeToRundownEvents(): EventSubscription[] {
     return [
       this.rundownEventObserver.subscribeToRundownActivation(this.activateBasicRundownFromEvent.bind(this)),
+      this.rundownEventObserver.subscribeToRundownRehearse(this.rehearsalBasicRundownFromEvent.bind(this)),
```

Comment:
Here it should be `rehearsal` and not `rehearse` because we subscribe to the noun, not the verb.

Reply:
Fixed